Makefile

```diff
@@ -103,7 +103,7 @@ ifeq (, $(shell which kustomize))
 	KUSTOMIZE_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$KUSTOMIZE_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/kustomize/kustomize/v3@v3.5.4 ;\
+	go get sigs.k8s.io/kustomize/kustomize/v3@v3.8.1 ;\
```

Comment:
Can we document why we need to pin another version? Just for the sake of history :)

Reply:
Ah, e2e is not working here due to this update :(

```
namespace/system created
customresourcedefinition.apiextensions.k8s.io/tenants.capsule.clastix.io created
clusterrole.rbac.authorization.k8s.io/capsule-proxy-role created
clusterrole.rbac.authorization.k8s.io/capsule-metrics-reader created
clusterrolebinding.rbac.authorization.k8s.io/capsule-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/capsule-proxy-rolebinding created
mutatingwebhookconfiguration.admissionregistration.k8s.io/capsule-mutating-webhook-configuration created
validatingwebhookconfiguration.admissionregistration.k8s.io/capsule-validating-webhook-configuration created
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
```

Reply:
seems like kustomize wasn't removed
```
sudo snap remove kustomize
  sudo snap remove kustomize
  shell: /bin/bash -e {0}
snap "kustomize" is not installed
```

and correct version of kustomize wasn't downloaded
```
kind load docker-image --nodes capsule-control-plane --name capsule quay.io/clastix/capsule:latest
Image: "quay.io/clastix/capsule:latest" with ID "sha256:7598fa7cfb473d786eb8c1d573308a7a5c4d2ac7b6e6a8632d94feeada3014d0" not present on node "capsule-control-plane"
make deploy
make[1]: Entering directory '/home/runner/work/capsule/capsule'
/home/runner/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
cd config/manager && /usr/local/bin/kustomize edit set image controller=quay.io/clastix/capsule:latest
/usr/local/bin/kustomize build config/default | kubectl apply -f -
namespace/system created
```

As there is no 
```
go: creating new go.mod: module tmp
go: downloading sigs.k8s.io/kustomize v3.8.1
```

And this version is needed because we need this PR https://github.com/kubernetes-sigs/kustomize/commit/74b45392fc1194e6003a1f1ab97dae96fb85a76d#diff-f058c1a370fe4d82b2864c52fe8e9554
In order to fix ordering when running `kustomize build`.

And version 3.8.2 had problems with https://github.com/operator-framework/operator-sdk/issues/3826

Reply:
But, maybe it was due to another issue (related to failurePolicy=Ignore), l will check it in a few hours :)

Reply:
Yep, this not needed, v3.5.4 is OK. Will rollback this change

Reply:
@prometherion 
I've rollbacked changes to Makefile, but there is still an error. Could you please take a look?